bunsen/bunsen-avro/src/main/java/com/cerner/bunsen/avro/converters/DefinitionToAvroVisitor.java

```diff
@@ -763,11 +766,6 @@ public HapiConverter<Schema> visitParentExtension(
       String extensionUrl,
       List<StructureField<HapiConverter<Schema>>> children) {
 
-    // Ignore extension fields that don't have declared content for now.
-    if (children.isEmpty()) {
```

Comment:
Hmm, is it safe to just drop this block? It seem to me that we rely on this `null` return value [here](https://github.com/google/fhir-data-pipes/blob/189671b4803d67f5f03b18156027e27d96c109fe/bunsen/bunsen-core/src/main/java/com/cerner/bunsen/definitions/StructureDefinitions.java#L240). Have you checked if without this block for an extension element without any `children`, the code still works fine?